### PR TITLE
Explain what .proto and .pb.go files are

### PR DESCRIPTION
**Pull request by pulkitsharma07**

### CODE REVIEW

**Comment by pulkitsharma07** [comment]
I signed it!

File: docs/quickstart/go.md [hunk 1]
```diff
@@ -64,7 +64,9 @@ Change to the example directory
 $ cd $GOPATH/src/google.golang.org/grpc/examples/helloworld
 ```
 
-gRPC services are defined in a proto file, which is used to generate a corresponding .pb.go. This file is already generated for the helloworld example code and can be found under this directory: `$GOPATH/src/google.golang.org/grpc/examples/helloworld/helloworld`
+gRPC services are defined in a `.proto` file, which is used to generate a corresponding `.pb.go` file. The `.pb.go` file is generated by compiling the `.proto` file using the protocol compiler: `protoc`.
+
+For the purpose of this example, `helloworld.pb.go` file has already been generated (by compiling `helloworld.proto`) and can be found under this directory: `$GOPATH/src/google.golang.org/grpc/examples/helloworld/helloworld`
```
Comment (LisaFC) [inline docs/quickstart/go.md]: Slight rephrase:

For the purpose of this example, the `helloworld.pb.go` file has already been generated (by compiling `helloworld.proto`), and can be found in this directory:

**Comment by LisaFC** [comment]
Thanks for this, tiny rewording and we're good to go!

**Comment by pulkitsharma07** [comment]
Done, please squash and merge :slightly_smiling_face:

**Pull request merged**